src/styles/index.scss

```diff
@@ -0,0 +1,28 @@
+@import './variables';
+
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: $mainFontSize;
+}
+
+body {
+  width: 100vw;
+  height: 100vh;
+  background-color: $primaryColor;
+}
+
+a:link,
+a:visited,
+a:active,
+a:hover {
+  color: $linkColor;
+  text-decoration: none;
+  text-transform: uppercase;
+}
```

Comment:
- [ ] Your project is good :clap: But there are some parts from  [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) you forgot :cry:  Please make sure to follow the design including Typography (font face, size, and weight) and Layout (composition and space between elements) as required for this project.  As a developer, it is very important to follow the original design. Consider the case where a client/company gives you a design, you need to provide and reproduce it exact product he wants in order to make it approve your work. In your future job, you will judge your ability to satisfy your clients or company. Please make it work :+1: 
- [ ]  For this project we expect you to create a mobile web application. Please make sure that your design is responsive and adapts to mobile resolutions. :smile: 
Please check the screenshots below to see the differences :arrow_down: 

#### Your design Home page
![Screenshot from 2022-10-22 20-30-25](https://user-images.githubusercontent.com/6615394/197361264-6f607f8a-6859-4353-9e54-1574cc05de59.png)

#### Your design Details page
![Screenshot from 2022-10-22 20-30-30](https://user-images.githubusercontent.com/6615394/197361275-b6d6d558-1756-48ba-9989-cdb21fa7eab3.png)

#### Your Expected Design Home page
![Screenshot from 2022-10-22 20-32-39](https://user-images.githubusercontent.com/6615394/197361391-39d2368a-5b1b-4fa1-81be-a9c464866dc9.png)

#### Your expected design Details page
![Screenshot from 2022-10-22 20-32-58](https://user-images.githubusercontent.com/6615394/197361423-88620fdb-d9ed-4071-a60b-681d0687cce9.png)



- [ ] Please make sure that you have your home and details page following the design guidelines and also the filter as defined in the design below :arrow_down: As a developer, it is very important to pay attention to details.  Kindly change the color theme that you chose for your design and kindly use one of the two provided color theme alternatives. This way, your project will match the Behance design and your project will be ready for approval. :smile: 

#### You can check the demo below :point_down: as an example
https://covid19-metrics-app-aka.netlify.app/

Reply:
@codecaiine , thank you for your kind review and for your keen suggestions, but I can't help but be disappointed at the idea that [this project](https://tourmaline-syrniki-7038a4.netlify.app/) passed without matching a single colour or style, while mine, even if it respects the design much better, is not good enough. I also see no filter feature in that project. I know that comparing is not an excuse, but thank you for making me fail the entire Microverse program - for this is the second time I'll have to repeat the week (and, so, I'll be thrown out of the program)! It's not like I can do anything in about 105 minutes.